airflow/www/static/js/dags.js

```diff
@@ -328,24 +325,32 @@ function taskStatsHandler(error, json) {
   });
 }
 
-if (encodedDagIds.has('dag_ids')) {
```

Comment:
I wrapped this block of code in a function and call it on page load (see the bottom of the file)